docs/setup/memfootprint-test/memfootprint-test-setup.md

```diff
@@ -0,0 +1,75 @@
+# Measuring memory footprint of EdgeCore
+
+## Why measuring memory footprint
+
+- This platform is also tagged for a light weighted edge computing deployment
+- To be able to be deployed over devices with less resources (for example, 256MB RAM)
+- It is required to know by deploying as many as possible pods, it showcases as much as less possible memory footprint
+
+## KPI's measured
+
+- %CPU
+- %Memory
+- Resident Set Size (RSS)
+
+## How to test
+
+After deployment and provisioning of kubeedge cloud and edge components in 2 VM's (supported and tested over ubuntu 16.04) respectively, start deploying pods from 0 to 100 in steps of 5. Keep capturing above KPI's using standard linux "ps" commands, after each step.
+
+### Test setup
+
+![Kubeedge TestSetup](./perftestsetup_diagram.PNG)  
+
+*Fig 1: Kubeedge TestSetup*
+
+### Creating a setup
+
+#### Requirements
+
+- Host machine's or VM's resource requirements can mirror the edge device of your choice
+- Resources used for above setup are 4 CPU, 8GB RAM and 200 GB Disk space. OS is Ubuntu 16.04.
+- Docker image used to deploy the pods in edge, needs to be created. The steps are:
+    1. Go to github.com/kubeedge/kubeedge/docs/setup/memfootprint-test/
+    2. Using the DockerFile available here and create docker image (`perftestimg:v1`).
+    3. Execute the docker command `sudo docker build --tag "perftestimg:v1" .`, to get the image.
+
+#### Installation
+
+* For kubeedge Cloud and Edge:
+
+    Please follow steps mentioned in Kubeedge README.md
+
+* For docker image:
+
+- Deploy docker registry to either edge on any VM or host which is reachable to edge. Follow the steps mentioned here: https://docs.docker.com/registry/deploying/
+- Create `perftestimg:v1` docker image on the above mentioned host
+- Then push this image to docker registry using "docker tag" and "docker push" commands (Refer: Same docker registry url mentioned above)
+  [Use this image's metadata in pod deployment yaml]
+
+### Steps
+
+1. Check edge node is connected to cloud. In cloud console/terminal, execute the below command
+
+```
+root@ubuntu:~/edge/pod_yamls# kubectl get nodes
+NAME                                   STATUS     ROLES    AGE     VERSION
+192.168.20.31                          Unknown    <none>   11s
+ubuntu                                 NotReady   master   5m22s   v1.14.0
+```
+
+2. On cloud, modify deployment yaml (github.com/kubeedge/kubeedge/docs/setup/memfootprint-test/perftestimg.yaml), set the image name and set spec.replica as 5
```

Comment:
Please do not place this yaml file under the /docs folder.

Reply:
@anvithks , @m1093782566 , @sids-b , As such i don't see any appropriate place holder for it. But What if we place the Dockerfile and yaml in kubeedge/edge/hack/memfootprint-test folder, what do you think ?

Reply:
@sids-b @m1093782566 please check if this works.  
@samy2019 You can take that call based on discussions with @sids-b and @m1093782566 .  
As long as you do not keep it under documentation it's all good.